crawler/src/crawler/block.ts

```diff
@@ -163,9 +172,21 @@ export const processBlocks = async (fromId: number, toId: number): Promise<numbe
   logger.info("Inserting events");
   await insertEvents(events);
 
+  // Transfers
+  logger.info("Extracting transfers");
+  let transfers = extrinsics
+    .filter(isExtrinsicTransfer)
+    .map(extrinsicBodyToTransfer);
+
+  logger.info("Compressing transfer and event accounts");
+  let allAccounts: AccountHead[][] = [];
+  allAccounts.push(...transfers.map(extractTransferAccounts));
+  allAccounts.push(...events.map(accountNewOrKilled))
+
   // Accounts
   logger.info("Extracting, compressing and dropping duplicate accounts");
-  let insertOrDeleteAccount = dropDuplicates(compress(events.map(accountNewOrKilled)), 'address')
+  let insertOrDeleteAccount = dropDuplicates(compress(allAccounts), 'address')
+    .filter(({address}) => address.length === 48);
```

Comment:
What is the purpose of `address.length == 48` here?

Reply:
Some extrinsics are paid but their signer is an empty address (like '') at the beginning of testnet. Therefore I'm labeling those accounts as deleted ones. But `deleted` address does not exist in the node so its balance can't be queried (throws an error) so I'm isolating only addresses that can be queried, which are the addresses with a length of 48.

Reply:
Or those paid extrinsics authors can be also labeled as `0x`

Reply:
cool makes sense :+1: